internal/pkg/manifest/http_test.go

```diff
@@ -158,6 +159,13 @@ func TestAlias_UnmarshalYAML(t *testing.T) {
 				// check memberwise dereferenced pointer equality
 				require.Equal(t, tc.wantedStruct.StringSliceOrString, r.Alias.StringSliceOrString)
 				require.Equal(t, tc.wantedStruct.AdvancedAliases, r.Alias.AdvancedAliases)
+
+				roundtrip, err := yaml.Marshal(tc.wantedStruct)
+				require.NoError(t, err)
+				fmt.Printf("re-marshalled form:\n%s\n", string(roundtrip))
```

Comment:
Is this here intentionally?

Reply:
This is intentional but it's possible that this and the other occurrences could be changed... it's really helpful to have them though when the test fails and even when it doesn't it helps me catch a lot of stylistic issues in the yaml just from inspection!

One example of stylistic issues is `omitempty` being forgotten, it's harmless (so it won't cause a test failure) but results in big structs with all-null fields and such in the resulting yaml, and being able to catch that via visual inspection helps.